test/UnitTests/Utils/TestServices.cpp

```diff
@@ -209,6 +223,9 @@ boolean TestAndCompareResults(const char* sTestPath, const char* test_suite, con
 	FILE* stream;
 	int fdInit;
 
+	// Parametrage de l'arret pour la memoire
+	//MemSetAllocIndexExit(3391);
+
```

Comment:
Should we keep this commented line ?

Reply:
Cela m'a été difficile de trouver l'endroit ou paramétrer cette ligne, puisque le framework GoogleTest ne semble pas avoir de main().
Je vais rajouter un commentaire.

Reply:
Ok